src/components/cards/WritableReviewCard.tsx

```diff
@@ -23,7 +24,7 @@ export default function WritableReviewCard({ estimation }: ReviewCardEstimations
         </div>
         <div className="flex lg:gap-[2.4rem] md:gap-[1.6rem] sm:gap-[1.2rem] items-center lg:w-[64rem] lg:h-[12.8rem] rounded-[0.6rem] border lg:px-[1.8rem] lg:py-[1.6rem] md:px-[0.8rem] md:py-[1.3rem] sm:px-0 sm:py-[1.3rem] bg-white border-line-100 lg:mt-[2.4rem] md:mt-[1.4rem] sm:mt-[1.4rem] lg:mb-[3.2rem] md:mb-[1.4rem] sm:mb-[1.4rem] ">
           <div className="lg:w-[9.6rem] lg:h-[9.6rem] md:w-[4.6rem] md:h-[4.6rem] sm:w-[4.6rem] sm:h-[4.6rem] relative ">
-            <Image src={estimation.driver.image} alt={estimation.driver.name} fill />
+            <Image src={estimation.driver.image ? estimation.driver.image : profile_default} alt={estimation.driver.name} fill />
```

Comment:
```suggestion
            <Image src={estimation.driver.image || profile_default} alt={estimation.driver.name} fill />
```
`estimation.driver.image가 존재하는 경우 estimation.driver.image를 보여주고, 아니면 profile_default를 보여주겠다`
라는 로직과 같은 로직이지만 OR문으로 변경할 경우 가독성이 좋아지고 코드가 간편해져서 보고 이해하기 더 쉬울 것 같다는 의견입니다!
아래의 다른 수정사항도 마찬가지입니다!

Reply:
넵 수정 완료했습니다